app/views/application/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+<!---Nav Bar--->
+  <header class="nav-bar">
```

Comment:
@aprildagonese instead of a header with a .nav-bar class this can just be a <nav> tag. Since <div> is a block element by default I would vote we remove the inner <div> tags with classes (like general-nav). Without those the <nav> will set up all the link_to's in <a> tags inline.

Reply:
@PeregrineReed Regarding the header with the .nav-bar class, do you just mean that we don't need the class because we can style the single <header> element? If that's what you mean, I have no problem with that. Regarding the inner tags defaulting to display: block, we can of course fix that during styling :) But given the conversation we had yesterday, about different nav links appearing in different places on the bar, it will be necessary to have them in separate containers. This made even more sense to me given that certain links will be appearing or disappearing based on the user's role, and we want the other links to remain as static as possible around those changes. Having them in separate containers is, IMO, the best way to achieve that.